app/assets/javascripts/foreman_openscap/openscap_proxy.js

```diff
@@ -1,52 +0,0 @@
-function updateOpenscapProxy(element){
```

Comment:
I guess https://github.com/theforeman/foreman_openscap/blob/master/app/views/compliance_hosts/_openscap_proxy.html.erb#L1 should be removed as well